examples/http_body/README.md

```diff
@@ -1,9 +1,15 @@
 ## http_body
 
-this example replaces the request body
+this example demnstrates how to perform operation on a request body like append/prepend/replace.
```

Comment:
typo: `demnstrates` -> `demonstrates`

Reply:
thanks! resolved in https://github.com/tetratelabs/proxy-wasm-go-sdk/pull/175